includes/class-wc-siftscience-dependencies.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * @package sift-for-woo
```

Comment:
in all the program it is siftsience

Reply:
I think we're using many different names for the package. I was not consistent when doing this.

I'll fix this in a separate pull request. #269